gobcore/model/gobmodel.json

```diff
@@ -3897,20 +3897,21 @@
               "formatted": "GOB.String"
             }
           },
-          "geboorteplaats": {
+          "heeft_geboorteplaats_nederland": {
+            "type": "GOB.Reference",
+            "description": "Code in tabel 33 van alle Nederlandse gemeenten",
+            "ref": "brp:gemeenten",
+            "secure": {
+              "bronwaarde": {
+                "type": "GOB.SecureString",
+                "level": 4
+              }
+            }
+          },
+          "heeft_geboorteplaats_buitenland": {
             "type": "GOB.JSON",
-            "description": "De gemeenteplaats van een nederlandse (BRK) gemeente of een locatieaanduiding van een plaats in het buitenland",
+            "description": "Een locatieaanduiding van een plaats in het buitenland",
             "attributes": {
-              "heeft_gemeentecode": {
```

Comment:
As discussed, please change to "heeft_geboorteplaats_buitenland" to be consistent with "heeft_geboorteplaats_nederland"

Reply:
Done